src/components/events/Events.js

```diff
@@ -200,30 +228,89 @@ const Events = () => {
           onButtonClick={loadGetEvents}
           onResetSearch={handleRefreshSearch}
         />
+        <br />
+        <Typography
+          style={{ color: "black", mb: 2, fontSize: 14, align: 'right' }}
+        >
+          Filters
+        </Typography>
+        <DropdownFilter
+          placeholder="Select the Status"
+          value={status}
+          onChange={handleStatus}
+          lists={statusList}
+        />
+        <br />
+        <NumberFilter
+          placeholder="Select the Maximum Number of Participants"
```

Comment:
Can you shrink the font size a teeny bit and make it gray? I feel like it looks a bit off relative to the search bar formatting. Maybe you can make the "Filters" text above it all to be bigger than it

Reply:
don't worry about styling, I can do it once its merged in